services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3RetentionPolicy.java

```diff
@@ -58,24 +60,27 @@ public S3RetentionPolicy(ObjectStoreAccess objectStoreAccess, DistributionServic
    * @param retentionDays the number of days, that files should be retained on S3.
    */
   public void applyRetentionPolicy(int retentionDays) {
-    List<S3Object> diagnosisKeysObjects = objectStoreAccess.getObjectsWithPrefix(api.getVersionPath() + "/"
-        + api.getVersionV1() + "/"
-        + api.getDiagnosisKeysPath() + "/"
-        + api.getCountryPath() + "/"
-        + api.getCountryGermany() + "/"
-        + api.getDatePath() + "/");
-    final String regex = ".*([0-9]{4}-[0-9]{2}-[0-9]{2}).*";
-    final Pattern pattern = Pattern.compile(regex);
+    supportedCountries.forEach(supportedCountry -> {
+      List<S3Object> diagnosisKeysObjects = objectStoreAccess.getObjectsWithPrefix(api.getVersionPath() + "/"
+            + api.getVersionV1() + "/"
+            + api.getDiagnosisKeysPath() + "/"
+            + api.getCountryPath() + "/"
+            + supportedCountry + "/"
+            + api.getDatePath() + "/");
+      final String regex = ".*([0-9]{4}-[0-9]{2}-[0-9]{2}).*";
+      final Pattern pattern = Pattern.compile(regex);
 
-    final LocalDate cutOffDate = TimeUtils.getUtcDate().minusDays(retentionDays);
+      final LocalDate cutOffDate = TimeUtils.getUtcDate().minusDays(retentionDays);
 
-    diagnosisKeysObjects.stream()
-        .filter(diagnosisKeysObject -> {
-          Matcher matcher = pattern.matcher(diagnosisKeysObject.getObjectName());
-          return matcher.matches() && LocalDate.parse(matcher.group(1), DateTimeFormatter.ISO_LOCAL_DATE)
-              .isBefore(cutOffDate);
-        })
-        .forEach(this::deleteDiagnosisKey);
+      diagnosisKeysObjects.stream()
+          .filter(diagnosisKeysObject -> {
+            Matcher matcher = pattern.matcher(diagnosisKeysObject.getObjectName());
+            return matcher.matches() && LocalDate.parse(matcher.group(1), DateTimeFormatter.ISO_LOCAL_DATE)
+                .isBefore(cutOffDate);
+          })
+          .forEach(this::deleteDiagnosisKey);
```

Comment:
I'm missing the real change here... it's just whitespace changes, right?

Reply:
Yes, this is because we moved the entire logic inside a forEach statement